src/components/ErrorNotification/ErrorNotification.tsx

```diff
@@ -0,0 +1,35 @@
+import { ErrorType } from '../../types/ErrorType';
+
+type Props = {
+  error: ErrorType;
+  onClose: () => void;
+};
+
+const errorMessages: Record<ErrorType, string> = {
+  load: 'Unable to load todos',
+  add: 'Unable to add a todo',
+  delete: 'Unable to delete a todo',
+  update: 'Unable to update a todo',
+  '': '', // пусто, если нет ошибки
+};
+
+export const ErrorNotification: React.FC<Props> = ({ error, onClose }) => {
+  if (!error) {
+    return null;
+  }
+
+  return (
+    <div
+      data-cy="ErrorNotification"
+      className="notification is-danger is-light has-text-weight-normal"
+    >
+      <button
+        data-cy="HideErrorButton"
+        type="button"
+        className="delete"
+        onClick={onClose}
+      />
+      {errorMessages[error]}
+    </div>
+  );
+};
```

Comment:
You should render Error component even if you don't have error. Just use hidden class for ErrorNotification to hide it.